apps/pie-docs/src/assets/styles/components/_centredPage.scss

```
@@ -18,10 +19,10 @@
 
     .c-centredPage-heading {
         margin-block-end: 0;
-        font-size: f.$font-heading-xxl-size--narrow * 1px;
+        font-size: #{p.font-size(--dt-font-heading-xxl-size--narrow)};
```

Is there a difference between these two syntaxes?
```suggestion
        @include p.font-size(--dt-font-heading-xxl-size--narrow);
```

---

nope, both should do the same thing. pie-css ships two ways of setting font-sizes. scss functions and mixins.